app/assets/stylesheets/views/comments.scss

```diff
@@ -269,3 +269,71 @@
     color: var(--base-90);
   }
 }
+
+.response-templates-container header {
+  padding-bottom: 1rem;
+}
+.personal-template-button,
+.moderator-template-button {
```

Comment:
refactor (blocking): I think we could simplify this CSS since it's pretty much just the styles from the crayons button components.

If you remove the `crayons-btn--outlined` class from the HTML element, you should be able to do this without needing any other classes to achieve the same result:

```css
.personal-template-button:not(.active),
.moderator-template-button:not(.active) {
   @extend .crayons-btn--outlined;
}
```

You'd need to add an import for that button class to the top of the file too:

```
@import '../components/buttons';
``` 

Aside from cutting down on lines, the benefit of this approach would be that if we make any changes to the crayons button component styles, they'd be immediately inherited here too 🙂